content/pages/sphinx.html

```diff
@@ -0,0 +1,294 @@
+    and get results broken out by code objects and paragraph. 
+    <strong>Never lose documentation again</strong>.
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-eye",
+        header="Never lose focus with link previews",
+        color="violet") %}
+
+    {# TODO: Add link preview to the `link previews` docs #}
+
+    {% markdown %}
+    Get instant context on every internal link in your docs.
+    With instant link previews, 
+    <strong>skim the content without opening *another* tab</strong>.
+    {% endmarkdown %}
+        {% endcall %}
+    {% endcall %}
+</section>
+
+<div class="ui very padded container">
+
+{{ basic.header_icon(
+  header="Join thousands of happy users",
+  subheader="",
+  icon="purple fa-message-lines",
+  icon_style="--fa-secondary-color: blueviolet;"
+) }}
+
+{# TODO: Turn these into testomonials. #}
+
+{#
+  Cards for an articles or case studies view
+
+  See https://fomantic-ui.com/views/card.html
+#}
+<div class="ui three link stackable cards">
+    <a href="https://flask.palletsprojects.com/en/2.2.x/" class="card">
+    <div class="ui small centered image">
+      <img src="{{ SITE_URL }}/images/flask-logo.png" alt="Flask logo">
+      </div>
+      <div class="content">
+        <div class="header">Flask</div>
+        <div class="meta">
+          Web Framework
+        </div>
+        <div class="description">
+            Flask is a Python web framework built with a small core and easy-to-extend philosophy.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.jupyter.org/en/latest/" class="card">
+    <div class="ui medium centered image">
+      <img src="{{ SITE_URL }}/images/jupyter-logo.png" alt="Jupyter logo">
+      </div>
+      <div class="content">
+        <div class="header">Jupyter</div>
+        <div class="meta">
+          Data Science Environment
+        </div>
+        <div class="description">
+            Jupyter Notebook is a web-based interactive computational environment for creating notebook documents.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.godotengine.org/en/stable/index.html" class="card">
+    <div class="ui small centered image">
+      <img src="{{ SITE_URL }}/images/godot-logo.png" alt="Godot Engine logo">
+    </div>
+      <div class="content">
+        <div class="header">Godot Engine</div>
+        <div class="meta">
+          Game Engine
+        </div>
+        <div class="description">
+          Godot Engine is a free and open source community-driven 2D and 3D game engine.
+        </div>
+      </div>
+    </a>
+</div>
+
+<div class="ui very padded basic vertical segment">
+  <div class="ui horizontal divider">Try it out</div>
+</div>
+
+<div class="ui horizontally container">
+  <div class="ui very padded centered grid">
+    <div class="middle aligned row">
+
+      <h2 class="ui huge header">
+        Everything you need, and nothing you don't.
```

Comment:
This header doesn't match the content below. This header sounds fairly vague and generic, but it's also saying "Read the Docs provides all the services you need to host your documentation in one place". The content doesn't agree with this, and is instead "how to host your Sphinx project on Read the Docs".

No strong preference here, but same energy but more specific benefit:

```suggestion
        Upgrade your documentation in minutes
```

Reply:
Also, the nesting of elements here is wrong. The heading should be outside the grid, and should be at least inside the `div.container`, and the grid shouldn't need a `div.row` class.

Reply:
I fixed the HTML structure. I edited the header for now so this doesn't get lost, but not sold on the edit copy.

Reply:
Moved to https://github.com/readthedocs/site-community/pull/152#discussion_r1016941390